spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/AtrousConvolution2D.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn.keras
+
+import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, TensorModule}
+import com.intel.analytics.bigdl.nn.{InitializationMethod, SpatialDilatedConvolution, Xavier, Zeros}
+import com.intel.analytics.bigdl.optim.Regularizer
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Shape
+
+import scala.reflect.ClassTag
+
+class AtrousConvolution2D[T: ClassTag](
```

Comment:
What's the expected order for this class? NCHW? or both?

Reply:
Only NCHW is supported. Will write this in the documentation.

Reply:
is there any parameters like dim_order here? or we also need to add this into the scala doc.

Reply:
Since we only support one format, it is strange to have a parameter `format`. I will add it into scala doc.

Reply:
Just check with Keras, there should be a `dim_ordering` in the parameter list. let's add it and throw exception if `channel-first` doesn't support for now.

Reply:
I think on scala side, we don't need such an option. It's really awkward that we provide an argument but it has to be only one value but no other choices. On python side, we can provide an argument just as Keras.

Reply:
Added dimOrdering